modules/mbedtls/Kconfig.psa

```diff
@@ -1,14 +1,355 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-config MBEDTLS_PSA_CRYPTO_CLIENT
+# This file was automatically generated by create_psa_files.py
+# from: ../../../modules/crypto/mbedtls/include/psa/crypto_config.h.
+# Do not edit it manually.
+
+config PSA_CRYPTO_CLIENT
 	bool
-	default y
-	depends on BUILD_WITH_TFM || MBEDTLS_PSA_CRYPTO_C
+	help
+	  Promptless symbol to state that there is a PSA crypto API provider
+	  enabled in the system. This allows to select desired PSA_WANT features.
+
+if PSA_CRYPTO_CLIENT
+
+config PSA_CRYPTO_ENABLE_ALL
+	bool "All PSA features"
```

Comment:
```suggestion
	bool "Enable all PSA crypto features"
```

Minor NIT

Reply:
I think I had issues where the CI was complaining about prompt starting with "Enable", so that's why I removed that part. I can add the "crypto" though